Scripts/Data/ConstrainedAxisFlags.cs

```diff
@@ -0,0 +1,17 @@
+﻿#if UNITY_EDITOR
+using System;
+
+namespace UnityEditor.Experimental.EditorVR
+{
+	/// <summary>
+	/// Which axes are constrained
+	/// </summary>
+	[Flags]
+	public enum ConstrainedAxis
```

Comment:
Either rename this enum to ConstrainedAxisFlags or the file to ConstrainedAxis.cs.